wowchemy/layouts/partials/widgets/contact.html

```diff
@@ -44,13 +51,14 @@
           <label class="sr-only" for="inputMessage">{{ i18n "contact_message" }}</label>
           <textarea name="message" class="form-control" id="inputMessage" rows="5" placeholder="{{ i18n "contact_message" | default "Message" }}" required></textarea>
         </div>
-        {{ if $use_netlify_form }}
-          <div class="d-none">
-            <label>Do not fill this field unless you are a bot: <input name="welcome-bot"></label>
-          </div>
-          {{ if $use_netlify_captcha }}
-            <div class="form-group" data-netlify-recaptcha="true"></div>
-          {{ end }}
+        <div class="d-none">
+          <label>Do not fill this field unless you are a bot: <input {{ if $use_netlify_captcha }}name="welcome-bot"{{ else if $use_formspree_form }}name="_gotcha"{{ end }}></label>
```

Comment:
Just use one input named `_gotcha` - remove the if/else logic here and change line 41 from `netlify-honeypot="welcome-bot"` to `netlify-honeypot="_gotcha"`